lib/scss/theme/_variables.scss

```diff
@@ -2,8 +2,7 @@
 $theme-highlight-color: #e6522c;
 
 // Headings
-$headings-color: $theme-highlight-color;
-$headings-alt-color: $theme-highlight-color;
+$headings-alt-color: #e6522c;
```

Comment:
Just a small nitpick that wouldn't change the end result, but we can use the `$theme-highlight-color` variable we declared for this